### DISCUSSION TITLE
Security vulnerabilities

### DISCUSSION THREAD

**Discussion by tarunkumar987** [pmodels/mpich#7132] [General]
Hi,
     We plan to move to the latest stable release(4.2.2?) from the current 3.2.1, however we would want to understand if there are any security vulnerabilities reported against this version. Considering security which would be a better release to move to? Is there way to understand more on the security issues reported against 4.x.x releases?

**Comment by hzhou** [comment]
We assume the compute nodes are secured against outside intrusion with firewall. There are no security measures against internal attack. That said, the worst an attack can do is to interrupt an MPI job.

The latest release does not have security issues reported vs. 3.2.1.
